src/test/java/seedu/address/model/person/PhoneTest.java

```diff
@@ -27,14 +27,15 @@ public void isValidPhone() {
         // invalid phone numbers
         assertFalse(Phone.isValidPhone("")); // empty string
         assertFalse(Phone.isValidPhone(" ")); // spaces only
-        assertFalse(Phone.isValidPhone("91")); // less than 3 numbers
+        assertFalse(Phone.isValidPhone("123456")); // less than 7 numbers
+        assertFalse(Phone.isValidPhone("1234567891234567")); // more than 15 numbers
         assertFalse(Phone.isValidPhone("phone")); // non-numeric
         assertFalse(Phone.isValidPhone("9011p041")); // alphabets within digits
         assertFalse(Phone.isValidPhone("9312 1534")); // spaces within digits
 
         // valid phone numbers
-        assertTrue(Phone.isValidPhone("911")); // exactly 3 numbers
+        assertTrue(Phone.isValidPhone("1234567")); // exactly 7 numbers
         assertTrue(Phone.isValidPhone("93121534"));
-        assertTrue(Phone.isValidPhone("124293842033123")); // long phone numbers
+        assertTrue(Phone.isValidPhone("123456789123456")); // exactly 15 numbers
```

Comment:
whats the rationale for having a range? international numbers? 

else everything looks good.